lectures/heavy_tails.md

```diff
@@ -516,11 +516,12 @@ for data, label, ax in zip(data_list, labels, axes):
 
     ax.loglog(x_grid, [eccdf(x, data) for x in x_grid], 
         'o', markersize=3.0, alpha=0.5, label=label)
-    ax.set_xlabel("log rank")
-    ax.set_ylabel("log size")
-
+    ax.set_ylabel("log prob")
+    
     ax.legend()
-
+    
+ax.set_xlabel("log value")
```

Comment:
@shlff , from this display it looks like the xlabel on 523 comes after `ax.legend()`.  Don't you want it to be on line 520 and indented the same as ylabel?

Reply:
Thanks @jstac . 

There are three stacked plots aligned by the same x-axis. With the above code I intended to keep only the x-axis label for the bottom plot to avoid repetition.

However, to add more clarity, I think it might be better to change it back. Please find the commit below.